Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/README.md

```diff
@@ -2175,4 +2179,236 @@ There is no context output for this command.
 You will be automatically redirected to a link with the following structure:
 >```REDIRECT_URI?code=AUTH_CODE&session_state=SESSION_STATE```
 >2. Copy the `AUTH_CODE` (without the `code=` prefix, and the `session_state` parameter)
-and paste it in your instance configuration under the **Authorization code** parameter.
\ No newline at end of file
+and paste it in your instance configuration under the **Authorization code** parameter.
+
+### msg-advanced-hunting
+
+***
+Advanced hunting is a threat-hunting tool that uses specially constructed queries to examine the past 30 days of event data in Microsoft Graph Security.
+To save result in context to 'Microsoft365Defender' as well, you can check the 'Microsoft 365 Defender context' checkbox in Instance Setting.
+
+#### Base Command
+
+`msg-advanced-hunting`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| query | Advanced hunting query. | Required | 
+| limit | Number of entries. Enter -1 for unlimited query, In case a limit also appears in the query, priority will be given to the query. | Optional | 
```

Comment:
```suggestion
| limit | Number of entries. Enter -1 for unlimited query. In case a limit also appears in the query, priority will be given to the query. | Optional | 
```